app/views/aspects/_aspect_dropdown.html.haml

```diff
@@ -0,0 +1,44 @@
+-# locals: selected_aspects. Note: all_aspects is a global in the ApplicationController
+.btn-group.aspect_dropdown
+  %button.btn.btn-default.dropdown-toggle{!current_user.getting_started? ? {"data-toggle" => "dropdown"} : |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.